<!--xml-->
<launch>
  <param name="capture_date" value="9Dec2014" />
    <param name="picked_date" value="9Dec2014" />
  <param name="show_images" value="1" />
  <param name="detect_method" value="ORB" />
  <param name="run_live" value="1" />
  <param name="filter" value="DIGITAL_FILTER" />

  <param name="threshold" value = "217" />
  
  <param name="erode" value="10" />
  
  <param name="dilate" value="9" />

  <param name="root_directory" value="/home/linaro/catkin_ws/src/icarus_rover_rc/" />
  <param name="template_image_name" value="Template1" />
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="log" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="960" />
    <param name="pixel_format" value="mjpeg" />c
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="run_targeting" pkg="icarus_rover_rc" type="run_targeting" output="screen">
   
  </node>
</launch>
